extension/extension.cpp

```diff
@@ -504,10 +504,13 @@ class UploadThread: public IThread
 
 #if defined _LINUX
 	bool UploadSymbolFile(const google_breakpad::CodeModule *module, const char *presubmitToken) {
+		if (log) fprintf(log, "UploadSymbolFile\n");
+		if (log) fflush(log);
+
 		auto debugFile = module->debug_file();
 		std::string vdsoOutputPath = "";
 
-		if (debugFile == "linux-gate.so") {
+		if (false && debugFile == "linux-gate.so") {
```

Comment:
Is this problematic everywhere or just linux64? It's a real pain to debug syscall crashes without this symbol file.

Reply:
Seems to be yes. At least I couldn't get the extension functional if at any point linux-gate.so is uploaded, couldn't really tell why, not sure how to investigate the issue either.

Reply:
No worries - it looks like it's got a different name on x64 as well, so this probably needs a fair bit of overhaul. Can leave it disabled and address it again in the future.